server/services/licenceService.ts

```diff
@@ -594,4 +595,8 @@ export default class LicenceService {
   async reviewWithoutVariation(licenceId: number): Promise<void> {
     return this.licenceApiClient.reviewWithoutVariation(licenceId)
   }
+
+  async getPrisonerDetail(nomsId: string, user: User): Promise<CaseloadItem> {
```

Comment:
I'd keep this on prisoner service.. it's part of the prisoner domain

Reply:
will move it to prisoner service in my next PR.